### PR TITLE
Fix updating teams

### PR DESCRIPTION
**Pull request by isuruf**
We should figure out a way to test these.

### CODE REVIEW

File: conda_forge_webservices/webapp.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def post(self):
                 comment = body['comment']['body']
                 commands.pr_comment(owner, repo_name, issue_num, comment)
 
-            if not pull_request and action in ['assigned', 'edited']:
+            if not pull_request and action in ['opened', 'edited', 'created', 'reopened']:
```
Comment (jakirkham) [inline conda_forge_webservices/webapp.py]: What was the `assigned` suppose to do?

Comment (isuruf) [reply conda_forge_webservices/webapp.py]: `assigned` is not relevant. The event is triggered when a issue is assigned to someone.

**Comment by jakirkham** [comment]
Side note: Do we rerun the team update check if we reopen an issue that asked for the team update to be run?

**Comment by isuruf** [comment]
@jakirkham, yes it is rerun

**Comment by jakirkham** [comment]
Are you testing this somewhere?

**Comment by isuruf** [comment]
https://github.com/conda-forge/constantly-feedstock/issues/2

**Pull request merged**